Assets/_Laz/Scripts/UserInterface/DebugBehaviour/DebugSceneChanger.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Laz
+{
+    public class DebugSceneChanger
+    {
+        private const string SCENE_NAME_DELIMITER = "/";
+        private const string SCENE_NAME_EXTENSION = ".unity";
+        
+        private List<string> _buildSceneNames = new List<string>();
+
+        public Action BuildSceneNamesChanged;
+
+        public List<string> BuildSceneNames => _buildSceneNames;
+
+        public void Initialize()
+        {
+            GetBuildSceneNames();
+            
+#if UNITY_EDITOR
+            UnityEditor.EditorBuildSettings.sceneListChanged += GetBuildSceneNames;
+#endif
+        }
+
+        public void CleanUp()
+        {
+#if UNITY_EDITOR
+            UnityEditor.EditorBuildSettings.sceneListChanged -= GetBuildSceneNames;
+#endif
+        }
+
+        public void ChangeScene(int buildIndex, Action onCallBack)
+        {
+            if (buildIndex < 0 || buildIndex >= _buildSceneNames.Count)
```

Comment:
if we default to build index 0, don't need to keep checking everywhere that buildIndex < 0.
Don't give it a chance to go -1 if you'll never give it -1 in the first place.

Reply:
Yeah but 0 is a valid build index since there is a scene at index 0